ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh

```diff
@@ -126,6 +126,8 @@ oc adm wait-for-stable-cluster
 
 # create new admin.kubeconfig
 oc config new-admin-kubeconfig > "${SHARED_DIR}/admin.kubeconfig"
+oc --kubeconfig="${SHARED_DIR}/admin.kubeconfig" whoami || true
```

Comment:
Looks like this line is not needed, as we have the check after the sleep ?

Reply:
we need to confirm if there is something wrong with this step after retry and the new kubeocnfig works fine.